scripts/lint-branch.sh

```diff
@@ -17,5 +17,5 @@ then
     echo "There is nothing to lint"
 else
     echo $LINES
-    NODE_OPTIONS="--max-old-space-size=8192" eslint $LINES
+    NODE_OPTIONS="--max-old-space-size=4096" eslint $LINES
```

Comment:
This change is to the linting shell script, so it will not effect production if we revert it.

Reply:
Yes, thats for github CI only
`lint-branch`